src/hooks/api.ts

```diff
@@ -0,0 +1,72 @@
+type Method = 'GET' | 'POST' | 'PUT' | 'DELETE';
+
+interface ApiOptions {
```

Comment:
Export `ApiOptions` and use it to create the `options` argument passed to `apiRequest`.

```suggestion
export interface ApiOptions {
```